client/js/autocompletion.js

```diff
@@ -311,8 +312,19 @@ function completeNicks(word, isFuzzy) {
 	return users.filter((w) => !w.toLowerCase().indexOf(word));
 }
 
+function getCommands() {
+	const clientCommands = Object.keys(commands).map((cmd) => `/${cmd}`);
+	const cmds = [...new Set(Array.from(constants.commands).concat(clientCommands))];
```

Comment:
move these outside of the function to make it easier for v8 to optimize

ref https://stackoverflow.com/a/58436106